lluelles-lexisnexis.csl

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-CA">
   <info>
-    <title>LexisNexis Québec (Guide Lluelles modifié, French - Canada)</title>
-    <id>http://www.zotero.org/styles/lluelles-lexisnexis</id>
-    <link href="http://www.zotero.org/styles/lluelles-lexisnexis" rel="self"/>
+    <title>Guide des références pour la rédaction juridique 7e édition (Notes complètes) (Guide Lluelles, No Ibid, French - Canada)</title>
+    <id>http://www.zotero.org/styles/lluelles-noibid</id>
```

Comment:
you'd also have to change the filename of this one to correspond to the ID. Instructions are the same as for any other edit.